components/is-analytics-solutions/org.wso2.carbon.sp.is.analytics.solutions/widgets/LineChart/src/LineChart.jsx

```diff
@@ -0,0 +1,148 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+
+import VizG from 'react-vizgrammar';
+import Widget from '@wso2-dashboards/widget';
+import {MuiThemeProvider, darkBaseTheme, getMuiTheme} from 'material-ui/styles';
+
+class LineChart extends Widget{
```

Comment:
Can't we rename the LineChart with its actual usage?
ex: AverageSessionDuration